Dockerfile

```diff
@@ -2,55 +2,31 @@ FROM python:3.9-slim
 
 ARG POETRY_VERSION="1.2.2"
 
-RUN groupadd --gid 1000 appuser \
-    && useradd --uid 1000 --gid 1000 -ms /bin/bash appuser
+WORKDIR /app
 
-RUN pip3 install --no-cache-dir --upgrade \
-    pip \
-    virtualenv
+COPY .env pyproject.toml poetry.lock README.md /app
+COPY ./src/ /app/src
+
+ENV PYTHONPATH=${PYTHONPATH}:${PWD}
 
 RUN apt-get update && apt-get install -y \
     build-essential \
     curl \
     software-properties-common \
     git \
-    ffmpeg \
-    sqlite3
+    ffmpeg
 
 RUN apt-get -y clean  \
     && rm -rf /var/lib/apt/lists/*
 
+RUN pip install .
 
-USER appuser
-WORKDIR /home/appuser
-
-RUN curl -sSL https://install.python-poetry.org |  \
-      python - --version ${POETRY_VERSION}
-
-ENV PATH="/home/appuser/.local/bin:$PATH"
-
-COPY .env pyproject.toml poetry.lock /home/appuser/
-
-RUN poetry cache clear pypi --all
-RUN poetry config virtualenvs.create true  \
-    && poetry config virtualenvs.in-project true  \
-    && poetry config experimental.new-installer true  \
-    && poetry install --no-root --without dev
-
-COPY ./src/ /home/appuser/src
-COPY README.md /home/appuser
-
-RUN poetry install --only-root
+WORKDIR /data
 
 EXPOSE 8501
 
-COPY run.sh /home/appuser
-
-USER root
-RUN mkdir /data && chown appuser: /data -R
-USER appuser
+ENV STREAMLIT_SERVER_ADDRESS=0.0.0.0
 
-RUN git config --global --add safe.directory '*'
-EXPOSE 8000
```

Comment:
I think that we would still want to expose port 8000 no?

Reply:
Don't know if needed but lets add it 👍